### ISSUE TITLE
easy import of common data

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse-components#1870]
Add functionality in the GUI for both admin and non-admin users to be able to easily import common assemblies and tracks with just a few clicks. Non-admins would just get these as session assemblies and session tracks, admins would be able to persist these to their configs if they are in the admin server.

Common assemblies and tracks like:
* hg19 and associated gene models from UCSC, Ensembl, etc
* same for hg38
* rat reference and annotations
* mouse
* cow
* etc

Implementation sketch:
* expand add-assembly dialog box to be able to add common assemblies from a JSON file on jbrowse.org that contains assembly configs that we keep up to date

**Comment by rbuels** [comment]
These needs a lot more design work

**Comment by garrettjstevens** [comment]
We discussed being able to choose assemblies/other data from the splash screen. Further discussion of user story is needed.

**Comment by cmdcolin** [comment]
Some thoughts for consideration



Pre-loaded config.json file with couple default things, which is what we currently have with desktop (a default config.json with hg19 and hg38) pointing to remote fasta files on our s3 bucket
- Pros: 
-- Fairly easy to deal with
- Cons: 
-- not easy to update url locations or add additional tracks to already-shipped-out desktop releases
-- users that don't care about other organisms might be annoyed to flip through those in an assembly list in the importform for example

Pre-configured snippets that we either ship with app or download from a location
- Pros
-- Fairly easy to deal with
-- can update if they are downloaded from a location
- Cons
-- after user has "applied the snippet" can't update then


Assemblies from a connections
- Pros
-- Some "infrastructure" for this already in codebase
- Cons
-- May be overly complicated for what we want
-- UI for connections can sometimes be hard to use
-- We'd have to use jbrowse 1 format, ucsc hub format, or make a jbrowse 2 connection format

**Comment by cmdcolin** [comment]
Another random note:
In JBrowse 1 desktop, it has the ability to "open a data directory" that can be a local directory with trackList.json, or it can be a remote URL directory with a trackList.json in it

- In JBrowse 1 in the local case, JBrowse 1 desktop can add tracks to that directory, which is nice
- In JBrowse 1 in the remote case, JBrowse 1 desktop can't add tracks which is not ideal

In JBrowse 2 desktop, a similar function could be cool
- In JBrowse 2 in the local case , it could add tracks to a local config.json in the local case.
- In JBrowse 2 in the remote case, it could use session tracks...or, some other alternative workflow to add tracks

**Comment by garrettjstevens** [comment]
Here is a possible user story I have considered:

- User downloads JBrowse Desktop and opens it for the first time
- User is prompted to provide a location, and a config file with no assemblies is created there for them
- Start screen appears to user
- On start screen, there's a very visible message that no assemblies are configured, with a button to click to configure some
- User clicks on button and is presented with an assembly setup wizard
- User can choose from one of a few JBrowse-provided assemblies, or from any of the assemblies in UCSC, or a custom assembly from (a) file(s)
- Once an assembly is chosen, users can also choose from a list of suggested tracks for that assembly, if there are any
- User can choose to point to these files remotely, or is given instructions on how to download them if needed for better performance
- User is then back at start screen, which now shows all configured assemblies and a button to add more, if wanted

**Comment by rbuels** [comment]
#2167 is related

**Comment by elliothershberg** [comment]
I talked with @cmdcolin today, and he brought up the idea (also discussed with @rbuels ) of having only the concept of session assemblies and tracks in desktop. This is a way to mitigate the problem of configs becoming enormous/unwieldy/breaking. Basically all data configuration would live in the session. Based on this, I had the following UI idea for the desktop start screen:

![image](https://user-images.githubusercontent.com/19295181/128412117-1a927ba3-23e2-4394-9751-1c757478f39e.png)

Basically, there would be two tabs at the top, with some visual indicator of which was selected. In the above example for "Choose organism", it would display different cards, with different assemblies. From an implementation perspective, clicking on these would load a pre-configured session snapshot, with data that we had curated and set up. One nice thing could be to have an expandable section on each card, giving some overview of what data was included.

This design is inspired by the ease of use of the [WashU epigenome browser](https://epigenomegateway.wustl.edu/) splash screen:

![image](https://user-images.githubusercontent.com/19295181/128412827-b0f1a175-ade8-42d5-b442-a02afdb3bbe2.png)

Clicking on one of the organisms, and then choosing assembly, it immediately launches a browser session for that organism.

If the user clicked on the "recent sessions" tab, they would see something like our existing list of sessions that shows up on the splash screen, and be able to click them to load them.

This idea basically is a combination of our existing splash screen with the WashU start screen to make the initial data load more seamless.

One downside is that we would lose the current buttons to directly open a linear genome view or SV inspector, but as @junjun-zhang mentioned, it could be good to make it easier to open those views from a blank screen anyways.
